portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewFields.testcase

```diff
@@ -1481,6 +1481,320 @@ definition {
 		}
```

Comment:
This is not the correct file where you have to create these tests. These automations are related to the Data Set filters, so you have to place the tests in the DataSetViewFilters.testcase file.